compiler/data/class-data.h

```diff
@@ -62,11 +64,14 @@ class ClassData : public Lockable {
   bool is_serializable{false};
   bool has_job_shared_memory_piece{false};
 
+  kphp_json::ClassTags *const kphp_json_tags{nullptr};
+
```

Comment:
It seems logical to place this field right after `const PhpDocComment *phpdoc`, even without a blank line

Reply:
ok, done